libr/core/cmd.c

```diff
@@ -96,6 +96,8 @@ static int bb_cmpaddr(const void *_a, const void *_b) {
 }
 
 static void cmd_debug_reg(RCore *core, const char *str);
+#include "cmd_helps.help"
```

Comment:
Please don't do this nonstandard code editors don't like non-standard extensions. Other problem is that now cmd_write.c depends on being included in a file which includes `cmd_helps.help` this also confuses code editors. I was hoping that after conversion to newshell we can get rid of including  cmd_x.c files and compile it properly.